database/engine/rrdengineapi.c

```diff
@@ -6,6 +6,8 @@ static struct rrdengine_instance default_global_ctx;
 
 int default_rrdeng_page_cache_mb = 32;
 int default_rrdeng_disk_quota_mb = 256;
+int computed_multidb_disk_quota_mb = 256;
```

Comment:
This global variable `computed_multidb_disk_quota_mb` is pretty much not used in a global scope.

It's initialized again inside `compute_multidb_diskspace()`, and it gets a value of `256` again inside `compute_multidb_diskspace()`, and its value could be returned by `compute_multidb_diskspace()` locally since it's used immediately afterwards.

Reply:
True !! `compute_multidb_diskspace` will now return the computed space

Reply:
You forgot to delete this from here and in `rrdengineapi.h`!

Reply:
Removed the variable `computed_multidb_disk_quota_mb` 

In rrdengineapi.h I have the `default_multidb_disk_quota_mb` which is needed

Reply:
Yeap, I'm not seeing clearly I guess!